### PR TITLE
Add additional logging details to Unity Adapters

### PR DESCRIPTION
**Pull request by PoojaChirp**

### CODE REVIEW

File: UnityAds/UnityAdsInterstitialCustomEvent.m [hunk 1]
```diff
@@ -37,6 +37,7 @@ - (void)requestInterstitialWithCustomEventInfo:(NSDictionary *)info {
     }
     if (self.placementId == nil) {
         [self.delegate interstitialCustomEvent:self didFailToLoadAdWithError:nil];
+        MPLogInfo(@" PlacementId is nil, place check the values set in the UI");
```
Comment (rsatrasala) [inline UnityAds/UnityAdsInterstitialCustomEvent.m]: typo - please

Comment (PoojaChirp) [reply UnityAds/UnityAdsInterstitialCustomEvent.m]: Nice catch! updated the PR.

File: UnityAds/UnityAdsInterstitialCustomEvent.m [hunk 2]
```diff
@@ -78,24 +79,28 @@ - (void)unityAdsReady:(NSString *)placementId
...
 {
     [self.delegate interstitialCustomEvent:self didFailToLoadAdWithError:nil];
+    MPLogInfo(@"Unity ads failed to load with an error message %@", message);
 }
 
 - (void) unityAdsDidStart:(NSString *)placementId
 {
     [self.delegate interstitialCustomEventWillAppear:self];
     [self.delegate interstitialCustomEventDidAppear:self];
+    MPLogInfo(@"Unity Ads to appear on your display");
```
Comment (rsatrasala) [inline UnityAds/UnityAdsInterstitialCustomEvent.m]: Display seems vague. Let's keep these messages consistent with the log messages of MoPub.

File: UnityAds/UnityAdsInterstitialCustomEvent.m [hunk 3]
```diff
@@ -106,6 +111,8 @@ - (void) unityAdsDidClick:(NSString *)placementId
 - (void)unityAdsDidFailWithError:(NSError *)error
 {
     [self.delegate interstitialCustomEvent:self didFailToLoadAdWithError:error];
+    MPLogInfo(@"Unity Ads failed with an error %@", error);
```
Comment (rsatrasala) [inline UnityAds/UnityAdsInterstitialCustomEvent.m]: error is an object and not a string. You would need to retrieve the message from error to display in the logs

File: UnityAds/UnityAdsRewardedVideoCustomEvent.m [hunk 4]
```diff
@@ -153,12 +153,14 @@ - (void)unityAdsDidError:(UnityAdsError)error withMessage:(NSString *)message
             break;
```
Comment (rsatrasala) [inline UnityAds/UnityAdsRewardedVideoCustomEvent.m]: There are some instances of NSLog. Modify them to MPLog

File: UnityAds/UnityAdsRewardedVideoCustomEvent.m [hunk 5]
```diff
@@ -179,6 +182,8 @@ - (void) unityAdsDidClick:(NSString *)placementId
 - (void)unityAdsDidFailWithError:(NSError *)error
 {
     [self.delegate rewardedVideoDidFailToLoadAdForCustomEvent:self error:error];
+    MPLogInfo(@"Unity ads failed to load with the following error: %@", error);
```
Comment (rsatrasala) [inline UnityAds/UnityAdsRewardedVideoCustomEvent.m]: Same feedback as above regarding error object

File: UnityAds/UnityAdsRewardedVideoCustomEvent.m [hunk 6]
```diff
@@ -179,6 +182,8 @@ - (void) unityAdsDidClick:(NSString *)placementId
...
 }
 
 @end
+
```
Comment (rsatrasala) [inline UnityAds/UnityAdsRewardedVideoCustomEvent.m]: Additional line at the end

File: UnityAds/UnityAdsInterstitialCustomEvent.m [hunk 7]
```diff
@@ -106,6 +111,8 @@ - (void) unityAdsDidClick:(NSString *)placementId
...
 }
 
 @end
+
```
Comment (rsatrasala) [inline UnityAds/UnityAdsInterstitialCustomEvent.m]: unwanted extra line. 
Minor one but these additional lines will show up in PRs unnecessarily. So best to remove them for future PRs.

File: UnityAds/UnityAdsInterstitialCustomEvent.m (hunk 3 above)
Comment (PoojaChirp) [reply UnityAds/UnityAdsInterstitialCustomEvent.m]: :thumbsup:

File: UnityAds/UnityAdsInterstitialCustomEvent.m (hunk 2 above)
Comment (PoojaChirp) [reply UnityAds/UnityAdsInterstitialCustomEvent.m]: :thumbsup:

File: UnityAds/UnityAdsRewardedVideoCustomEvent.m (hunk 4 above)
Comment (PoojaChirp) [reply UnityAds/UnityAdsRewardedVideoCustomEvent.m]: :thumbsup:

File: UnityAds/UnityAdsRewardedVideoCustomEvent.m (hunk 5 above)
Comment (PoojaChirp) [reply UnityAds/UnityAdsRewardedVideoCustomEvent.m]: :thumbsup:

File: UnityAds/UnityAdsRewardedVideoCustomEvent.m (hunk 6 above)
Comment (PoojaChirp) [reply UnityAds/UnityAdsRewardedVideoCustomEvent.m]: :thumbsup:

File: UnityAds/UnityAdsInterstitialCustomEvent.m (hunk 7 above)
Comment (PoojaChirp) [reply UnityAds/UnityAdsInterstitialCustomEvent.m]: :thumbsup:

**Comment by rsatrasala** [comment]
Closing this and these changes will be added when we improve all logging at the same time.

**Pull request closed without merging**